tools/codegen/core/experiments_compiler.py

```diff
@@ -600,20 +597,32 @@ def GenExperimentsBzl(self, output_file):
                 file=B,
             )
 
-            bzl_to_tags_to_experiments = sorted(
-                (self._bzl_list_for_defaults[default], tags_to_experiments)
-                for default, tags_to_experiments in bzl_to_tags_to_experiments.items()
-                if self._bzl_list_for_defaults[default] is not None
-            )
-
             print(file=B)
-            print("EXPERIMENTS = {", file=B)
-            for key, tags_to_experiments in bzl_to_tags_to_experiments:
-                print('    "%s": {' % key, file=B)
-                for tag, experiments in sorted(tags_to_experiments.items()):
-                    print('        "%s": [' % tag, file=B)
-                    for experiment in sorted(experiments):
-                        print('            "%s",' % experiment, file=B)
-                    print("        ],", file=B)
+            if mode == "test":
+                print("TEST_EXPERIMENTS = {", file=B)
+            else:
+                print("EXPERIMENTS = {", file=B)
+
+            print(bzl_to_tags_to_experiments["windows"].items())
+            for platform in self._platforms_define.keys():
+                bzl_to_tags_to_experiments_platform = sorted(
+                    (self._bzl_list_for_defaults[default], tags_to_experiments)
+                    for default, tags_to_experiments in bzl_to_tags_to_experiments[
+                        platform
+                    ].items()
+                    if self._bzl_list_for_defaults[default] is not None
+                )
+                print('    "%s": {' % platform, file=B)
```

Comment:
Once the tests come back online, I think the linter is going to complain about inconsistent quote types in this file.

Reply:
Not sure. Sanity was passing before. I'll keep an eye on it.